service/build.gradle

```diff
@@ -365,6 +365,7 @@ dependencies {
   implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-joda', version: '2.15.3'
 
   implementation group: 'net.minidev', name: 'json-smart', version: '2.5.0'
+  implementation group: 'com.nimbusds', name: 'nimbus-jose-jwt', version: '9.37.3'
```

Comment:
Guess this is for the new CVE just now. Looks like the Spring 2.7 upgrade fixes this without having to add this extra dependency explicitly. Wonder if we should maybe suppress for now and remove when the Spring upgrade is released?

Reply:
sounds good! Removed it